coordinator/src/main/java/org/apache/uniffle/coordinator/metric/CoordinatorMetrics.java

```diff
@@ -96,6 +98,23 @@ public static void updateDynamicGaugeForRemoteStorage(String storageHost, double
     }
   }
 
+  public static void addDynamicGaugeForUser(String user) {
```

Comment:
Should we merge this method to `addQuotaMetrics`?

Reply:
I don't think it is necessary here, because it needs to introduce `MetricsManager` into `QuotaManager`, right?

Reply:
OK. I misread it